docs/changelog.md

```diff
@@ -2,6 +2,9 @@
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) format.
 
+## [0.7.1] - 2024-xx-xx
+### Added
+- added `portable` flag to allow for easier sharing of reports.
```

Comment:
```suggestion
- added `portable` flag to allow easier sharing of reports. 
```

Reply:
This is tricky. I believe 'allow for' is actually what we want here, but I'm happy to see evidence to the contrary.

But I think I'm going to change this as:
`added portable flag to pipestat summarize to create a shareable version of the html report`